src/web/lib/hakuneko/lang/en.json

```diff
@@ -0,0 +1,181 @@
+{
```

Comment:
If the localization really is supposed to be used globally (for the HakuNeko engine and all frontends), then all keys for the modern frontend should be placed inside
```json
{
  "frontend" : {
    "modern": {
      // keys go here ...
    }
  }
}
```